tests/proxy_models/tests.py

```diff
@@ -105,6 +105,17 @@ def test_proxy_included_in_ancestors(self):
         with self.assertRaises(Person.MultipleObjectsReturned):
             StatusPerson.objects.get(id__lt=max_id + 1)
 
+    def test_multiple_abc(self):
```

Comment:
No as `User` and `Person` are registered to the global `apps` instance. The creation of the class is supposed to fail anyway so the `apps` cache won't get polluted.